components/Header.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+const Headline = styled.h1`
```

Comment:
if this is going to be a Header, you could also thing about using the header element to be more semantic:

```suggestion
const Headline = styled.header`
```